see list of users on team 164484578

justgarcia5 opened the pull request.

Commit:
edit user form and implemented testing

Commit:
added user feature and tests

JustinNealHatch committed:
Merge branch 'master' into features/edit-a-user-164704330

Commit:
updated features/user_spec test and passed

justgarcia5 committed:
Merge branch 'master' into features/edit-a-user-164704330

JustinNealHatch committed:
added test and in progress for routes

JustinNealHatch committed:
Added team_members logic

kevinyu05 committed:
added partial form, switched logic on users index

kevinyu05 committed:
fixed syntax

kevinyu05 committed:
fixed css on nav link & logo

kevinyu05 committed:
fixed routing issue, WIP

Commit:
implemented test for team leader seeing team members feature

jasononaquest commented on app/views/layouts/application.html.erb:
```diff
@@ -14,8 +14,10 @@
           <% if current_user.is_admin? %>
             <%= link_to "User List", users_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'users'}" %>
-            <%= link_to "Initiatives", root_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'initiatives'} " %>
+            <% elsif current_user.team_members.present? %>
```
This should probably be outdented to the same level as the `if`.

justgarcia5 replied:
We resolved this issue

jasononaquest commented on app/views/users/index.html.erb:
```diff
@@ -1,6 +1,8 @@
 <div class="wrapper">
   <div class="add-user-btn">
+    <% if current_user.is_admin %>
     <%= link_to 'Add User', new_user_path, class: 'btn-primary w-button' %>
```
It can be helpful to indent blocks within an `if-else`.

justgarcia5 replied:
This issue was resolved

jasononaquest commented on config/routes.rb:
```diff
@@ -2,6 +2,7 @@
 
 Rails.application.routes.draw do
   devise_for :users, :controllers => { :sessions => "sessions" }
+  get '/users/team_members', to: 'users#team_members', as: 'team'
```
Is there a path that makes more sense than `/users/team_members` for this route? Using `as` suggests you think `/team` makes more sense. How could you rewrite this without the `as` to achieve that?

justgarcia5 replied:
The path was changed to /team

kevinyu05 committed:
cleaned up logic to represent similar logic in index

Commit:
resolved PR comments

jasononaquest reviewed (approved):
LGTM

hyosunko committed:
fixed error on edit user/initiative
add test for edit user/initiative
extract user form from users/index
Coauthored_by: Kevin Yu<kevinyu05@gmail.com>
Coauthered_by: Justin Garcia<justgarcia5@gmail.com>

hyosunko committed:
fix css error

hyosunko committed:
fixed user css
fixed merge conflict
change user partial form
